test/integration_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/src-d/gitquery"
+	"github.com/src-d/gitquery/internal/function"
+	"github.com/stretchr/testify/require"
+	sqle "gopkg.in/src-d/go-mysql-server.v0"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+func TestIntegration(t *testing.T) {
+	engine := sqle.New()
+	pool := gitquery.NewRepositoryPool()
+	require.NoError(t, pool.AddDir(".")) // TODO: add repositories for testing
+
+	engine.AddDatabase(gitquery.NewDatabase("testing", &pool))
+	function.Register(engine.Catalog)
+
+	for _, query := range queries {
+		t.Run(query.name, func(t *testing.T) {
+			schema, rowIter, err := engine.Query(query.statement)
+			if query.expectedErr {
+				require.Error(t, err)
+			} else {
+				require.NoError(t, err)
+				checkSchema(t, schema, query.expectedSchema)
+				checkIter(t, rowIter, query.expectedRows)
+			}
+		})
+	}
+}
+
+func checkSchema(t *testing.T, schema, expected sql.Schema) {
+	// TODO: add expected schema to the queries and check it
+	require.Nil(t, expected)
+}
+
+func checkIter(t *testing.T, rowIter sql.RowIter, expected int) {
+	// TODO: add expected number of rows to the queries and check it
+	require.Zero(t, expected)
```

Comment:
change to `require.Equals(expected,count)` to make it fails until the queries will be implemented